nmodl/dsl.py

```diff
@@ -39,6 +39,6 @@ def load_example(example):
     """
     path = files("nmodl") / RESOURCE_DIR / example
     if path.exists():
-        return str(path)
+        return path.read_text()
```

Comment:
The docstring for "Returns:" seems to describe something different, i.e. return a path, not the contents of the file.

Reply:
Ah yes, I forgot to change the docstring in the last revision.
While we're on the subject of the return type, the docs for `pkg_resources.resource_string` say:

> Return the specified resource as bytes. The resource is read in binary fashion, such that the returned string contains exactly the bytes that are stored in the resource.

and while I can just replace the call with `read_bytes`, does it make any tangible difference for later parsing by the C++ module?

Reply:
My guess is that pybind11 is responsible for converting string or byte to `std::string` correctly.

Reply:
Indeed, the reference says it will always use UTF-8: https://pybind11.readthedocs.io/en/stable/advanced/cast/strings.html
so I think it's a non-issue.